rivus/utils/prerun.py

```diff
@@ -0,0 +1,45 @@
+"""
+    Collection of small rivus related util functions
+    
+    In use to avoid multiple solutions of the same
+    problem.
```

Comment:
I don't really get what you mean, but I convert it to pep8 conform and extend it